commit 82e16ebf642c6d703cba61ebaaca4fa7b9ef2cca (refs/changes/29/829/1)
Author: Chayeon Hong <chayeon.hong@samsung.com>
Date:   2016-11-01T16:37:54+09:00 (2 years, 11 months ago)
    
    joyent/smartos-live#673 EBUSY should be handled in smartos env also

diff --git a/src/img/node_modules/rimraf/rimraf.js b/src/img/node_modules/rimraf/rimraf.js
index eb96c46a..a79162f3 100644
--- a/src/img/node_modules/rimraf/rimraf.js
+++ b/src/img/node_modules/rimraf/rimraf.js
@@ -43,7 +43,7 @@ function rimraf (p, options, cb) {
   var busyTries = 0
   rimraf_(p, options, function CB (er) {
     if (er) {
-      if (isWindows && (er.code === "EBUSY" || er.code === "ENOTEMPTY") &&
+      if ((er.code === "EBUSY" || er.code === "ENOTEMPTY") &&
           busyTries < exports.BUSYTRIES_MAX) {
         busyTries ++
         var time = busyTries * 100
